ddtrace/opentracer/span.py

```diff
@@ -87,7 +88,10 @@ def set_baggage_item(self, key, value):
         :rtype: Span
         :return: itself for chaining calls
         """
-        self.context.set_baggage_item(key, value)
+        new_ctx = self.context.with_baggage_item(key, value)
+        self.set_tag(key, value)
```

Comment:
we should not set this as tags because `baggage` != `tag`. Our platform should not receive baggage values right now.